+ source /root/mugen/testcases/cli-test/libguestfs/common/common_libguestfs.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Thu Jun 22 16:23:48 2023 - INFO  - Start environmental preparation.
+ libguestfs_pre_test
+ libguestfs_pre_test_no_install
+ DNF_INSTALL 'libguestfs virt-manager qemu libvirt openssl-devel numactl numactl-devel libcap-ng-devel traceroute iperf3 python3-paramiko edk2-devel qemu-guest-agent virt-install.noarch'
+ pkgs='libguestfs virt-manager qemu libvirt openssl-devel numactl numactl-devel libcap-ng-devel traceroute iperf3 python3-paramiko edk2-devel qemu-guest-agent virt-install.noarch'
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs 'libguestfs virt-manager qemu libvirt openssl-devel numactl numactl-devel libcap-ng-devel traceroute iperf3 python3-paramiko edk2-devel qemu-guest-agent virt-install.noarch' --node 1 --tempfile ''
+ tmpfile2='Last metadata expiration check: 0:02:01 ago on Thu 22 Jun 2023 04:22:10 PM CST.
Package libguestfs-1:1.49.5-2.oe2303.riscv64 is already installed.
Package virt-manager-4.1.0-1.oe2303.noarch is already installed.
Package qemu-10:6.2.0-68.oe2303.riscv64 is already installed.
Package libvirt-6.2.0-52.oe2303.riscv64 is already installed.
Package python3-paramiko-2.11.0-1.oe2303.noarch is already installed.
No match for argument: edk2-devel
Error: Unable to find a match: edk2-devel'
+ '[' -z '' ']'
+ tmpfile='Last metadata expiration check: 0:02:01 ago on Thu 22 Jun 2023 04:22:10 PM CST.
Package libguestfs-1:1.49.5-2.oe2303.riscv64 is already installed.
Package virt-manager-4.1.0-1.oe2303.noarch is already installed.
Package qemu-10:6.2.0-68.oe2303.riscv64 is already installed.
Package libvirt-6.2.0-52.oe2303.riscv64 is already installed.
Package python3-paramiko-2.11.0-1.oe2303.noarch is already installed.
No match for argument: edk2-devel
Error: Unable to find a match: edk2-devel'
++ uname -i
+ test riscv64X == aarch64X
+ systemctl start libvirtd
+ mkdir -p /home/kvm/images
+ chown root:root /home/kvm/images
+ chmod 755 /home/kvm/images
++ arch
+ LOG_INFO 'get riscv64 qcow2 libguestfs pkg'
+ message='get riscv64 qcow2 libguestfs pkg'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'get riscv64 qcow2 libguestfs pkg'
Thu Jun 22 16:24:13 2023 - INFO  - get riscv64 qcow2 libguestfs pkg
++ arch
+ '[' '!' -f ./common/openEuler-20.03-LTS-SP3-riscv64.qcow2 ']'
++ arch
++ arch
++ arch
+ wget https://repo.huaweicloud.com/openeuler/openEuler-20.03-LTS-SP3/virtual_machine_img/riscv64/openEuler-20.03-LTS-SP3-riscv64.qcow2.xz --no-check-certificate -O ./common/openEuler-20.03-LTS-SP3-riscv64.qcow2.xz
--2023-06-22 16:24:13--  https://repo.huaweicloud.com/openeuler/openEuler-20.03-LTS-SP3/virtual_machine_img/riscv64/openEuler-20.03-LTS-SP3-riscv64.qcow2.xz
Resolving repo.huaweicloud.com (repo.huaweicloud.com)... 119.167.243.54, 119.167.243.57, 119.167.243.51, ...
Connecting to repo.huaweicloud.com (repo.huaweicloud.com)|119.167.243.54|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2023-06-22 16:24:14 ERROR 404: Not Found.

++ arch
+ xz -d ./common/openEuler-20.03-LTS-SP3-riscv64.qcow2.xz
xz: ./common/openEuler-20.03-LTS-SP3-riscv64.qcow2.xz: File format not recognized
++ arch
+ cp ./common/openEuler-20.03-LTS-SP3-riscv64.qcow2 /home/kvm/images/openEuler-20.03-LTS-SP3.qcow2
cp: cannot stat './common/openEuler-20.03-LTS-SP3-riscv64.qcow2': No such file or directory
+ virt-install --name openEuler-2003 --ram 2048 --vcpus=2 --disk path=/home/kvm/images/openEuler-20.03-LTS-SP3.qcow2,bus=virtio,format=qcow2 --network=bridge:virbr0 --force --import --autostart --noautoconsole --graphics none
/root/mugen/testcases/cli-test/libguestfs/common/common_libguestfs.sh: line 23: virt-install: command not found
+ virsh destroy openEuler-2003
error: failed to get domain 'openEuler-2003'

+ echo hello
+ virt-copy-in -a /home/kvm/images/openEuler-20.03-LTS-SP3.qcow2 a.txt /etc
/usr/bin/virt-copy-in: access: /home/kvm/images/openEuler-20.03-LTS-SP3.qcow2: No such file or directory
+ unset LIBGUESTFS_BACKEND
+ export LIBGUESTFS_DEBUG=1 LIBGUESTFS_TRACE=1
+ LIBGUESTFS_DEBUG=1
+ LIBGUESTFS_TRACE=1
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Thu Jun 22 16:24:16 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Jun 22 16:24:16 2023 - INFO  - Start to run test.
+ virt-sysprep --password root:password -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 33: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --password failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --password failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --password failed'
+ LOG_ERROR 'Check virt-sysprep --password failed'
+ message='Check virt-sysprep --password failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --password failed'
Thu Jun 22 16:24:17 2023 - ERROR - Check virt-sysprep --password failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 34'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 34'
Thu Jun 22 16:24:17 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --password-crypto md5 -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 35: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --password-crypto failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --password-crypto failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --password-crypto failed'
+ LOG_ERROR 'Check virt-sysprep --password-crypto failed'
+ message='Check virt-sysprep --password-crypto failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --password-crypto failed'
Thu Jun 22 16:24:18 2023 - ERROR - Check virt-sysprep --password-crypto failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 36'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 36'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 36'
Thu Jun 22 16:24:18 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 36
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep -q -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 37: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep -q failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep -q failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep -q failed'
+ LOG_ERROR 'Check virt-sysprep -q failed'
+ message='Check virt-sysprep -q failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep -q failed'
Thu Jun 22 16:24:19 2023 - ERROR - Check virt-sysprep -q failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 38'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 38'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 38'
Thu Jun 22 16:24:19 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 38
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --root-password password -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 39: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --root-password failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --root-password failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --root-password failed'
+ LOG_ERROR 'Check virt-sysprep --root-password failed'
+ message='Check virt-sysprep --root-password failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --root-password failed'
Thu Jun 22 16:24:20 2023 - ERROR - Check virt-sysprep --root-password failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 40'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 40'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 40'
Thu Jun 22 16:24:21 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 40
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --run-command cat -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 41: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep ---run-command failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep ---run-command failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep ---run-command failed'
+ LOG_ERROR 'Check virt-sysprep ---run-command failed'
+ message='Check virt-sysprep ---run-command failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep ---run-command failed'
Thu Jun 22 16:24:21 2023 - ERROR - Check virt-sysprep ---run-command failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 42'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 42'
Thu Jun 22 16:24:22 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --scriptdir /etc/tmp -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 43: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --scriptdir failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --scriptdir failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --scriptdir failed'
+ LOG_ERROR 'Check virt-sysprep --scriptdir failed'
+ message='Check virt-sysprep --scriptdir failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --scriptdir failed'
Thu Jun 22 16:24:22 2023 - ERROR - Check virt-sysprep --scriptdir failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 44'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 44'
Thu Jun 22 16:24:23 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --scrub /etc/a.txt -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 45: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --scrub failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --scrub failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --scrub failed'
+ LOG_ERROR 'Check virt-sysprep --scrub failed'
+ message='Check virt-sysprep --scrub failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --scrub failed'
Thu Jun 22 16:24:23 2023 - ERROR - Check virt-sysprep --scrub failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 46'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 46'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 46'
Thu Jun 22 16:24:24 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 46
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --selinux-relabel -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 47: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --selinux-relabel failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --selinux-relabel failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --selinux-relabel failed'
+ LOG_ERROR 'Check virt-sysprep --selinux-relabel failed'
+ message='Check virt-sysprep --selinux-relabel failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --selinux-relabel failed'
Thu Jun 22 16:24:24 2023 - ERROR - Check virt-sysprep --selinux-relabel failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 48'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 48'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 48'
Thu Jun 22 16:24:25 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 48
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --timezone Europe/London -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 49: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --timezone failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --timezone failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --timezone failed'
+ LOG_ERROR 'Check virt-sysprep --timezone failed'
+ message='Check virt-sysprep --timezone failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --timezone failed'
Thu Jun 22 16:24:26 2023 - ERROR - Check virt-sysprep --timezone failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 50'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 50'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 50'
Thu Jun 22 16:24:26 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 50
+ '[' 0 -eq 1 ']'
+ return 0
+ virt-sysprep --touch /etc/a.txt -d openEuler-2003
oe_test_libguestfs_virt-sysprep_04.sh: line 51: virt-sysprep: command not found
+ CHECK_RESULT 127 0 0 'Check virt-sysprep --touch failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check virt-sysprep --touch failed'
+ exit_mode=0
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check virt-sysprep --touch failed'
+ LOG_ERROR 'Check virt-sysprep --touch failed'
+ message='Check virt-sysprep --touch failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check virt-sysprep --touch failed'
Thu Jun 22 16:24:27 2023 - ERROR - Check virt-sysprep --touch failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_libguestfs_virt-sysprep_04.sh line 52'
+ message='oe_test_libguestfs_virt-sysprep_04.sh line 52'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_libguestfs_virt-sysprep_04.sh line 52'
Thu Jun 22 16:24:27 2023 - ERROR - oe_test_libguestfs_virt-sysprep_04.sh line 52
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End of the test.'
+ message='End of the test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End of the test.'
Thu Jun 22 16:24:28 2023 - INFO  - End of the test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 10
+ test 10 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Jun 22 16:24:28 2023 - ERROR - failed to execute the case.
+ exit 10
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Jun 22 16:24:29 2023 - INFO  - Start to restore the test environment.
+ libguestfs_post_test
+ virsh start openEuler-2003
error: failed to get domain 'openEuler-2003'

+ virsh destroy openEuler-2003
error: failed to get domain 'openEuler-2003'

+ virsh undefine openEuler-2003 --nvram
error: failed to get domain 'openEuler-2003'

+ unset LIBGUESTFS_DEBUG LIBGUESTFS_TRACE
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z Last metadata expiration check: 0:02:01 ago on Thu 22 Jun 2023 04:22:10 PM CST.
Package libguestfs-1:1.49.5-2.oe2303.riscv64 is already installed.
Package virt-manager-4.1.0-1.oe2303.noarch is already installed.
Package qemu-10:6.2.0-68.oe2303.riscv64 is already installed.
Package libvirt-6.2.0-52.oe2303.riscv64 is already installed.
Package python3-paramiko-2.11.0-1.oe2303.noarch is already installed.
No match for argument: edk2-devel
Error: Unable to find a match: edk2-devel ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile 'Last metadata expiration check: 0:02:01 ago on Thu 22 Jun 2023 04:22:10 PM CST.
Package libguestfs-1:1.49.5-2.oe2303.riscv64 is already installed.
Package virt-manager-4.1.0-1.oe2303.noarch is already installed.
Package qemu-10:6.2.0-68.oe2303.riscv64 is already installed.
Package libvirt-6.2.0-52.oe2303.riscv64 is already installed.
Package python3-paramiko-2.11.0-1.oe2303.noarch is already installed.
No match for argument: edk2-devel
Error: Unable to find a match: edk2-devel'
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 173, in <module>
    exitcode = rpm_remove(args.node, args.pkgs, args.tempfile)
  File "/root/mugen/libs/locallibs/rpm_manage.py", line 144, in rpm_remove
    with open(tmpfile, "r") as f:
OSError: [Errno 36] File name too long: 'Last metadata expiration check: 0:02:01 ago on Thu 22 Jun 2023 04:22:10 PM CST.\nPackage libguestfs-1:1.49.5-2.oe2303.riscv64 is already installed.\nPackage virt-manager-4.1.0-1.oe2303.noarch is already installed.\nPackage qemu-10:6.2.0-68.oe2303.riscv64 is already installed.\nPackage libvirt-6.2.0-52.oe2303.riscv64 is already installed.\nPackage python3-paramiko-2.11.0-1.oe2303.noarch is already installed.\nNo match for argument: edk2-devel\nError: Unable to find a match: edk2-devel'
+ '[' 0 -ne 0 ']'
+ rm -rf /home/kvm/images/ a.txt './*.xz' 'initramfs*' output 'p2v*' 'pidfile*' 'pre-*' 'qemu-4.2.0*' test1.img 'vmlinuz*' 'file*'
+ LOG_INFO 'Finish to restore the test environment.'
+ message='Finish to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish to restore the test environment.'
Thu Jun 22 16:24:33 2023 - INFO  - Finish to restore the test environment.
